.eslintrc.js

```diff
@@ -2,23 +2,21 @@ module.exports = {
   env: {
     browser: true
   },
-  extends: "eslint:recommended",
+  extends: ["plugin:prettier/recommended"],
```

Comment:
This makes eslint use the prettier styling and config and as such we dont need to add all the ignore rules below like we had before.

Reply:
nice